documentation/modules/configuring/con-config-mirrormaker2-producers-consumers.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// assembly-config-mirrormaker2.adoc
+
+[id='con-mirrormaker-producers-consumers-{context}']
+= Connector producer and consumer configuration
+
+[role="_abstract"]
+MirrorMaker 2.0 connectors uses internal producers and consumers.
+If needed, you can configure these producers and consumers to override the default settings. 
```

Comment:
Maybe we need to say something about config exceptions?

Reply:
This statement appears to be fine. I think it is safe to leave.